cmd/kubeseal/main.go

```diff
@@ -51,24 +51,25 @@ const (
 
 var (
 	// TODO: Verify k8s server signature against cert in kube client config.
-	certURL        = flag.String("cert", "", "Certificate / public key file/URL to use for encryption. Overrides --controller-*")
-	controllerNs   = flag.String("controller-namespace", metav1.NamespaceSystem, "Namespace of sealed-secrets controller.")
-	controllerName = flag.String("controller-name", "sealed-secrets-controller", "Name of sealed-secrets controller.")
-	outputFormat   = flag.StringP("format", "o", "json", "Output format for sealed secret. Either json or yaml")
-	outputFileName = flag.StringP("sealed-secret-file", "w", "", "Sealed-secret (output) file")
-	inputFileName  = flag.StringP("secret-file", "f", "", "Secret (input) file")
-	dumpCert       = flag.Bool("fetch-cert", false, "Write certificate to stdout. Useful for later use with --cert")
-	allowEmptyData = flag.Bool("allow-empty-data", false, "Allow empty data in the secret object")
-	printVersion   = flag.Bool("version", false, "Print version information and exit")
-	validateSecret = flag.Bool("validate", false, "Validate that the sealed secret can be decrypted")
-	mergeInto      = flag.String("merge-into", "", "Merge items from secret into an existing sealed secret file, updating the file in-place instead of writing to stdout.")
-	raw            = flag.Bool("raw", false, "Encrypt a raw value passed via the --from-* flags instead of the whole secret object")
-	secretName     = flag.String("name", "", "Name of the sealed secret (required with --raw and default (strict) scope)")
-	fromFile       = flag.StringSlice("from-file", nil, "(only with --raw) Secret items can be sourced from files. Pro-tip: you can use /dev/stdin to read pipe input. This flag tries to follow the same syntax as in kubectl")
-	sealingScope   ssv1alpha1.SealingScope
-	reEncrypt      bool // re-encrypt command
-	unseal         = flag.Bool("recovery-unseal", false, "Decrypt a sealed secrets file obtained from stdin, using the private key passed with --recovery-private-key. Intended to be used in disaster recovery mode.")
-	privKeys       = flag.StringSlice("recovery-private-key", nil, "Private key filename used by the --recovery-unseal command. Multiple files accepted either via comma separated list or by repetition of the flag. Either PEM encoded private keys or a backup of a json/yaml encoded k8s sealed-secret controller secret (and v1.List) are accepted. ")
+	certURL            = flag.String("cert", "", "Certificate / public key file/URL to use for encryption. Overrides --controller-*")
+	controllerNs       = flag.String("controller-namespace", metav1.NamespaceSystem, "Namespace of sealed-secrets controller.")
+	controllerName     = flag.String("controller-name", "sealed-secrets-controller", "Name of sealed-secrets controller.")
+	controllerPortName = flag.String("controller-port-name", "", "Port name of sealed-secrets controller.")
```

Comment:
It seems this parameter isn't used in the `main()` function when invoking the `run` function... Also.. It seems we're already assuming that the SealedSecret service has a port with name "http" in functions such as `validateSealedSecret` or `reEncryptSealedSecret`, see:

- https://github.com/bitnami-labs/sealed-secrets/blob/main/cmd/kubeseal/main.go#L316
- https://github.com/bitnami-labs/sealed-secrets/blob/main/cmd/kubeseal/main.go#L350

Couldn't do a similar assumption in `openCertCluster`?